### ISSUE TITLE
Closing lightbox opened from inside pop-up also closing pop-up

### ISSUE THREAD

**Issue by euene** [elementor/elementor#14005]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description

When a lightbox is opened after clicking on a link inside a pop-up, and then closing the lightbox, both the lightbox and the pop-up will be closed.

The desired UX should be that the pop-up will remain opened after closing the lightbox.

It looks like both components share the same/similar close event, but they should be separate.

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

WordPress 5.6.2

Only plug-ins active:
Elementor 3.1.1
Elementor Pro 3.1.0

Active theme:
Hello Elementor 2.3.1

Server:
PHP 7.4.15
Apache/2.4.46
MariaDB 10.5.8

**Comment by Johndo36** [comment]
Yes, encountered the same problem on multiple client websites. Would love to see this one adressed in a future update. This would improve the UX a lot.

**Comment by iAmoghJain** [comment]
This is actually isn't a bug. You just have to alter a setting in the pop-up closing which is in the Elementor itself.

Here's the key to do it:
Navigate to Elementor Pop-ups in the WP Dashboard -> Edit with Elementor -> Click the Gear Icon in down left (first icon in toolbar where you click Update/Publish) -> Click Advanced Pop-up Settings -> Again click Advanced -> Turn on "Prevent Closing on Overlay" -> Update the changes!

And you're pretty much done and good to go! This method works perfect for all my Pop-ups...Hope it helps for you too.

If you want a Premium Professional Website for yourself contact me.

Amogh Jain
Founder, PiX Motion Studios

**Comment by JxxIT** [comment]
> This actually isn't a bug. You just have to alter a setting in the pop-up closing which is in the Elementor itself.
> 
> Here's the key to do it: Navigate to Elementor Pop-ups in the WP Dashboard -> Edit with Elementor -> Click the Gear Icon in down left (first icon in toolbar where you click Update/Publish) -> Click Advanced Pop-up Settings -> Again click Advanced -> Turn on "Prevent Closing on Overlay" -> Update the changes!
> 
> And you're pretty much done and good to go! This method works perfect for all my Pop-ups...Hope it helps for you too.
> 
> If you want a Premium Professional Website for yourself contact me.
> 
> Amogh Jain Founder, PiX Motion Studios

Thanks! Works for me.
